css/src/components/avatar.css

```diff
@@ -123,3 +123,16 @@
 .Avatar-presence--away {
   background: var(--secondary-dark);
 }
+
+.Avatar-status {
+  top: calc(-1 * var(--spacing-s));
+  right: calc(-1 * var(--spacing-s));
+  width: var(--spacing-l);
+  height: var(--spacing-l);
+  border-radius: 50%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: absolute;
+  cursor: pointer;
+}
```

Comment:
Please add overflow hidden as the content should not go beyond the status indicator.